server/service/src/sync/translations/invoice_line.rs

```diff
@@ -269,8 +268,16 @@ impl SyncTranslation for InvoiceLineTranslation {
             r#type: line_type,
             number_of_packs,
             note,
-            inventory_adjustment_reason_id,
-            return_reason_id: None, // TODO
+            inventory_adjustment_reason_id: match invoice.r#type {
+                InvoiceType::InventoryAddition | InvoiceType::InventoryReduction => {
+                    option_id.clone()
+                }
+                _ => None,
+            },
+            return_reason_id: match invoice.r#type {
+                InvoiceType::CustomerReturn | InvoiceType::SupplierReturn => option_id,
+                _ => None,
+            },
```

Comment:
Couldn't do an overall match and have to check separately here since it kept giving me FK constraint even though the FK was correct....